src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )}
+
+      <p className="Person__partner">
+        {!person.isMarried
+          ? 'I am not married'
+          : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
let's avoid using nested ternary operators
create variable and put there part of the expression